generate-wasm-examples/generate_wasm_examples-webgpu.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+git init bevy-main
+cd bevy-main
+git remote add origin https://github.com/mockersf/bevy
+git pull --depth=1 origin example-runner
```

Comment:
Once https://github.com/bevyengine/bevy/pull/8561 will be merged
```suggestion
git remote add origin https://github.com/bevyengine/bevy
git pull --depth=1 origin main
```